frontend/src/data/saga.js

```diff
@@ -52,3 +53,17 @@ function* watchDownloadImage(action) {
     yield put(singleRequest.error());
   }
 }
+
+function* watchGetWords(action) {
+  try {
+    let words;
+    const updateWithSuccess = async () => {
+      words = await firebaseInteractor.getWords();
+    };
+    yield call(updateWithSuccess);
+    yield put(singleRequest.getWordsSuccess({ words }));
+  } catch (error) {
+    console.log(error);
```

Comment:
i think it's aight not to put console.log(error) here since yield put sorta has it. But either way is fine

Reply:
I had this when debugging, will remove lol

Reply:
Forgot to remove. I will make a big pr removing all console.logs prob next week in case something breaks and I need to try to debug firebase issues.